OrbitLinuxTracing/TracerThread.cpp

```diff
@@ -409,6 +410,21 @@ bool TracerThread::OpenThreadNameTracepoints(const std::vector<int32_t>& cpus) {
       &thread_name_tracepoint_ring_buffer_fds_per_cpu, &ring_buffers_);
 }
 
+void TracerThread::InitThreadStateVisitor() {
+  thread_state_visitor_ = std::make_unique<ThreadStateVisitor>();
+  thread_state_visitor_->SetListener(listener_);
+  event_processor_.AddVisitor(thread_state_visitor_.get());
+}
+
+bool TracerThread::OpenThreadStateTracepoints(const std::vector<int32_t>& cpus) {
+  absl::flat_hash_map<int32_t, int> thread_state_tracepoint_ring_buffer_fds_per_cpu;
+  return OpenFileDescriptorsAndRingBuffersForAllTracepoints(
+      {{"sched", "sched_switch", &sched_switch_ids_},
```

Comment:
Aren't you missign the wakeup_new/new_task/sched_waking events? Am I missing something?

Reply:
Okay, then I misunderstood those. I thought there are cases were sched_waking are used, and others were sched_wakeup are use.d